common/brave_content_client.h

```diff
@@ -6,15 +6,15 @@
 #ifndef BRAVE_COMMON_BRAVE_CONTENT_CLIENT_H_
 #define BRAVE_COMMON_BRAVE_CONTENT_CLIENT_H_
 
-#include "chrome/common/chrome_content_client.h"
+#include "content/public/common/content_client.h"
 
-class BraveContentClient : public ChromeContentClient {
+class BraveContentClient : public content::ContentClient {
```

Comment:
We want to inherit `ChromeContentClient` and additionally overrides two more methods from it.
This change will break our code.

Reply:
FYI, You can refer https://github.com/brave/brave-core/blob/master/patches/components-sync-BUILD.gn.patch to know what @bridiver told.

Reply:
@simonhong Thanks for correcting me. Fixed.